Commands/Help.js

```diff
@@ -1,7 +1,7 @@
 //this file will house all commands attached to the play command
 
-const help = ()=> {
-  console.log('Help Command was called')
+const help = (message)=> {
+  message.channel.send('Thank you for using the SoundWave Bot. \n The current features that you can use are \n "!play": to play any sound in the directory \n "!list": to list all songs in the directory \n "!help": well that is what got you here, so type again to see this list of commands')
 }
 
 export default help 
```

Comment:
Nitpick: this wasn't part of this PR, but I'd consider putting empty lines at the end of your files.